__Main__.ipynb

```diff
@@ -0,0 +1,904 @@
+{
```

Comment:
<p>Not a useful plot. Look closely though, why are there repeating patterns in the x-direction?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AI-Core-Assignments/project-1-Farhad1353/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
<p>My expectation was to see a repeating patterns.</p><p>At 1st stage of the Random Forest,I did implement nested loop(with three loops which changes the hyper-parameters. </p><p>So as at some point when we have the same figure for the two inner loops but different value for the outer loop I expect to see some sort of repeating pattern.</p><p>I understand, I need to learn a smarter approach to find a better accuracy score. </p><p>So, I'm not justifying my method is good. I am only explaining why based on the method I've chosen we should expect a repeating pattern.</p>